gulpfile.js

```diff
@@ -247,6 +250,23 @@ gulp.task("link-bugs", function (done) {
     exec(bin("linkify-bugs.sh"), [path.join(SOURCE_DIR, "_posts")], done);
 });
 
+gulp.task('lint', function() {
+    return gulp.src(path.join(LINT_DIR, "**", "*.html"))
+        .pipe(htmllint());
+});
+
+gulp.task('checklinks', function(cb) {
+  crawler.crawl('http://localhost:3000/')
```

Comment:
A server should be running for this task to succeed, but a server should not be started every time this task runs. Let's instead encode this dependency by adding a statement like `console.log("Make sure the server is running on " + SERVE_URL + "before running this task.")` to the `'checklinks'` task.